ui/src/main/js/global/channels/jira/server/JiraServerGlobalConfiguration.js

```diff
@@ -93,6 +93,7 @@ const JiraServerGlobalConfiguration = ({ csrfToken, readonly }) => {
                     onChange={FieldModelUtilities.handleChange(formData, setFormData)}
                     errorName={FieldModelUtilities.createFieldModelErrorKey(JIRA_SERVER_GLOBAL_FIELD_KEYS.configurePlugin)}
                     errorValue={errors[JIRA_SERVER_GLOBAL_FIELD_KEYS.configurePlugin]}
+                    successBox
```

Comment:
Does this make the successbox appear? Is that how we originally had Jira server and cloud?

Reply:
I recall we used to show a checkbox after it installed the plugin.  Nothing in the dynamic side of things is setting the successBox in the java code so I will remove it.